dbms/src/Flash/Pipeline/Pipeline.cpp

```diff
@@ -72,7 +72,7 @@ PipelineExecGroup Pipeline::toExecGroup(Context & context, size_t concurrency)
 {
     assert(!plan_nodes.empty());
     PipelineExecGroupBuilder builder;
-    for (auto it = plan_nodes.rbegin(); it != plan_nodes.rend(); ++it)
+    for (auto it = plan_nodes.begin(); it != plan_nodes.end(); ++it)
```

Comment:
```suggestion
    for (const auto & plan_node : plan_nodes)
```
Also need to modify `Pipeline::addPlanNode` and `Pipeline::addGetResultSink`